### PR TITLE
Make use of matrix in tests.yml

### PR DESCRIPTION
**Pull request by szepeviktor**
# 👨‍💻

### CODE REVIEW

**Comment by szepeviktor** [comment]
Dyslexia rulez!

**Comment by szepeviktor** [comment]
@mlocati Please do squash my commits.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -40,14 +44,14 @@ jobs:
       - name: Build
         run: ./bin/unipoints build
       - name: Check built assets
-        if: startsWith(matrix.os, 'windows') == false
+        if: ${{ startsWith(matrix.os, 'windows') == false }}
```
Comment (szepeviktor) [inline .github/workflows/tests.yml]: This is Explicit Viktor running at half throttle 🚂
Full throttle would be [quoting all YAML strings](https://github.com/szepeviktor/byte-level-care/blob/master/.github/workflows/back-end.yml).

Comment (mlocati) [reply .github/workflows/tests.yml]: I find quoted strings somehow less readable

Comment (szepeviktor) [reply .github/workflows/tests.yml]: Yes, they are less readable.
But YAML is for machines, not for hoomans :)
So I do everything to make YAML safe.
And use `yamllint` to enforce those.

Comment (mlocati) [reply .github/workflows/tests.yml]: Also, in `if` cases the `${{ ... }}` are optional, right?
When there are two alternatives, I always prefer the less verbose syntax (like `[]` vs `arrray()` in PHP).

Comment (mlocati) [reply .github/workflows/tests.yml]: Yaml is read both by humans and machines (I wrote it, not chatgpt 😉 )

Comment (szepeviktor) [reply .github/workflows/tests.yml]: > I always prefer the less verbose syntax

Okay-okay I cannot really talk to developers. My single goal is having long error-free years.

Comment (szepeviktor) [reply .github/workflows/tests.yml]: I let you finish/close/merge this PR.

Comment (szepeviktor) [reply .github/workflows/tests.yml]: I haven't come up with 4-space indentation and non-indented lists!
Those take months to get used to.

Comment (szepeviktor) [reply .github/workflows/tests.yml]: I have 103 0️⃣0️⃣0️⃣ 0️⃣0️⃣0️⃣ reasons.

image

**Pull request merged**